README.md

```diff
@@ -32,29 +39,36 @@ If the installation was successful, you should be able to run the following comm
 
 If you need to update `npm`, you can make it using `npm`!
 
-    $ npm install npm -g
+    npm install npm -g
 
 ---
 
 ## Install
 
-    $ git clone https://github.com/DynamoDS/NotificationCenter
-    $ cd NotificationCenter
-    $ npm install --force
-
+    git https://github.com/DynamoDS/NotificationCenter
+    cd NotificationCenter
+    npm install
```

Comment:
The `--force` is also on purpose because of a known issue of dependency conflict on HIG notification flyout, were you able to build locally without it?